src/workers/parser/getSymbols.js

```diff
@@ -212,7 +213,17 @@ function extractSymbol(path: SimplePath, symbols) {
       return;
     }
 
-    if (t.isProperty(path.parent)) {
+    // e.g. myFunc({c} = {c: 2})
+    const inAssignment = t.isAssignmentPattern(path.getAncestor(3));
+
+    // e.g myFunc({ a })
+    const isFunctionParamShorthand =
+      isObjectShorthand(path.parent) && t.isCallExpression(path.getAncestor(3));
+
+    if (
+      t.isProperty(path.parent) &&
+      !(inAssignment || isFunctionParamShorthand)
+    ) {
```

Comment:
correct me if i'm wrong, but this does not seem to give us the correct expression to evaluate. the tooltip just shows `undefined`

Reply:
Scratch dat!
![](http://g.recordit.co/q4suXW00Xy.gif)
I notice we are missing some use cases e.g 
- line 24 `frameActor` in `{ frameActor: frameId }`
- line 28 `b ` in `const foo = ({ b })` etc

Reply:
Hey @atwalg2 it is the behaviour we want ... but it seems like it only works for particular cases. 

If you don't mind i'll make some updates.

Reply:
I definitely do not mind. I could use some help.